src/parser/jobs/pld/modules/Requiescat.tsx

```diff
@@ -171,10 +167,7 @@ export class Requiescat extends BuffWindow {
 	override output() {
 		return <Fragment>
 			<Message>
-				<Trans id="pld.requiescat.table.note">Each of your <DataLink status="REQUIESCAT" /> windows should contain 4 spells at minimum to maintain the alignment of your rotation.
-				Most of the time, a window should consist of 4 casts of <DataLink action="HOLY_SPIRIT" /> or <DataLink action="HOLY_CIRCLE" /> and end with a cast of <DataLink action="CONFITEOR" />.
-				However, under some circumstances, it is useful to drop one <DataLink action="HOLY_SPIRIT"/> per minute in order to better align your rotation with buffs or mechanics.
-				If you don't have a specific plan to do this, you should aim for 4 casts of <DataLink action="HOLY_SPIRIT" /> per <DataLink status="REQUIESCAT" /> window.</Trans>
+				<Trans id="pld.requiescat.table.note">Each of your <DataLink status="REQUIESCAT" /> windows should contain 5 spells, consisting of 4 casts of <DataLink action="HOLY_SPIRIT" /> or <DataLink action="HOLY_CIRCLE" /> and endING with a cast of <DataLink action="CONFITEOR" />.</Trans>
```

Comment:
Unexpected capitalization of `endING`
```suggestion
				<Trans id="pld.requiescat.table.note">Each of your <DataLink status="REQUIESCAT" /> windows should contain 5 spells, consisting of 4 casts of <DataLink action="HOLY_SPIRIT" /> or <DataLink action="HOLY_CIRCLE" /> and ending with a cast of <DataLink action="CONFITEOR" />.</Trans>
```

(Don't forget to `yarn extract` after this change)